tests/test_voa.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+import voap
+
+def test_copula():
+    true_copula = np.array(
+        [
+            -0.1505847, -0.2742042, -0.3742406, -0.4714045, -0.5741693, 0.69006556, 0.5741693, 0.4714045, 0.3742406, 
+            0.2742042, 0.1505847, -0.2742042, -0.4993070, -0.6814663, 1.1200012, 0.8286734, 1.25656172, 1.0455226, 
+            0.8583951, 0.6814663, 0.4993070, 0.2742042, -0.3742406, -0.6814663, -0.9300817, 0.4485395, 0.1078143, 
+            0.20579830, 1.4269545, 1.1715584, 0.9300817, 0.6814663, 0.3742406, -0.4714045, -0.8583951, 0.4485395, 
+            1.4395893, 0.9643376, 0.80237742, 1.7974341, 1.4757296, 1.1715584, 0.8583951, 0.4714045, -0.5741693, 
+            -1.0455226, 0.1078143, 0.9643376, 0.4270426, 0.05847053, 0.8811133, 0.4165482, -0.1078143, 1.0455226, 
+            0.5741693, 0.6900656, -0.5863955, 0.2057983, 0.8023774, 1.3448223, 0.63245553, 1.3448223, 0.8023774, 
+            0.2057983, 1.2565617, 0.6900656, 0.5741693, 1.0455226, 1.4269545, 1.7974341, 2.1892691, 1.34482230, 
+            1.7351985, 0.9643376, 0.1078143, 0.8286734, -0.5741693, 0.4714045, 0.8583951, 1.1715584, 1.4757296, 
+            1.7974341, 0.80237742, 0.9643376, 1.4395893, 0.4485395, 1.1200012, -0.4714045, 0.3742406, 0.6814663, 
+            0.9300817, 1.1715584, 1.4269545, 1.71498585, 1.6425832, 2.0686374, 0.8705564, 1.5173982, -0.3742406, 
+            0.2742042, 0.4993070, 0.6814663, 0.8583951, 1.0455226, 1.25656172, 0.8286734, 1.1200012, 1.5173982, 
+            2.1858551, -0.2742042, 0.1505847, 0.2742042, 0.3742406, 0.4714045, 0.5741693, 0.69006556, -0.5741693, 
+            -0.4714045, -0.3742406, -0.2742042, -0.1505847,
+        ]
+    )
+    true_copula = true_copula.reshape((11, 11))
+
+    r = np.array(range(1, 11))
+    s = np.array([3, 6, 2, 9, 4, 1, 7, 5, 8, 10])
```

Comment:
one letter variable names usually don't make much sense - unless they are named in the paper same way?

Reply:
The names come from the paper and the equations that are there.